diff --git a/sfweb2lead_webform/src/Plugin/WebformHandler/SalesforceWebToLeadPostWebformHandler.php b/sfweb2lead_webform/src/Plugin/WebformHandler/SalesforceWebToLeadPostWebformHandler.php
index 6e877d4393..9b8d563d7a 100644
--- a/sfweb2lead_webform/src/Plugin/WebformHandler/SalesforceWebToLeadPostWebformHandler.php
+++ b/sfweb2lead_webform/src/Plugin/WebformHandler/SalesforceWebToLeadPostWebformHandler.php
@@ -189,7 +189,7 @@ protected function getRequestData($state, WebformSubmissionInterface $webform_su
 
     // Allow modification of data by other modules.
     $event = new Sfweb2leadWebformEvent($salesforce_data, $this, $webform_submission);
-    $this->eventDispatcher()->dispatch(Sfweb2leadWebformEvent::SUBMIT, $event);
+    $this->eventDispatcher()->dispatch($event, Sfweb2leadWebformEvent::SUBMIT);
 
     return $event->getData();
   }